runtime/lua/vim/lsp/client.lua

```diff
@@ -715,7 +715,7 @@ end
 --- @param params table LSP request params.
 --- @param timeout_ms integer? Maximum time in milliseconds to wait for
 ---                                a result. Defaults to 1000
---- @param bufnr integer Buffer handle (0 for current).
+--- @param bufnr? integer Buffer handle (0 for current).
```

Comment:
You need to run `make doc` to add this to the respective text file

Reply:
Sorry, I forgot and force-pushed now.